src/org/zaproxy/zap/extension/ascanrules/TestPathTraversal.java

```diff
@@ -373,7 +421,7 @@ public void scan(HttpMessage msg, String param, String value) {
                     || errorMatcher.find()) {
 
                 if (log.isDebugEnabled()) {
-                    log.debug("It IS possible to check for local file Path Traversal on the url filename on ["
+                    log.debug("It is possible to check for local file Path Traversal on the url filename on ["

```

Comment:
@psiinon @thc202 you good with that?

I know formatting and  standardization is on the big TODO list :+1: